geode-core/src/main/java/org/apache/geode/management/internal/cli/result/model/ResultModel.java

```diff
@@ -178,7 +181,51 @@ public ResultModel createCommandProcessingError(String message) {
   public ResultModel createError(String message) {
     addInfo().addLine(message);
     setStatus(Result.Status.ERROR);
+    return this;
+  }
 
+  public ResultModel createInfo(String message) {
+    addInfo().addLine(message);
+    setStatus(Result.Status.OK);
```

Comment:
Once an error has been created (and the status set to error) we shouldn't allow the status to be reverted.